cake/build-settings.cake

```diff
@@ -68,16 +63,6 @@ public class BuildSettings
             "Run mock-assembly-x86.dll under .NET 4.6.2",
             $"src/TestData/mock-assembly-x86/bin/{Configuration}/net462/mock-assembly-x86.dll",
             MockAssemblyExpectedResult(1));
-        NetCore31X86Test = new PackageTest(
-            "NetCore31X86Test",
-            "Run mock-assembly-x86.dll under .NET Core 3.1",
-            $"src/TestData/mock-assembly-x86/bin/{Configuration}/netcoreapp3.1/mock-assembly-x86.dll",
-            MockAssemblyExpectedResult(1));
```

Comment:
You dropped a net core 3.1 test, was that deliberate?

Reply:
Unintentional... see following comment.